libkbfs/folder_branch_ops.go

```diff
@@ -92,8 +92,8 @@ type fboMutexLevel mutexLevel
 
 const (
 	fboMDWriter fboMutexLevel = 1
-	fboHead                   = 2
-	fboBlock                  = 3
+	fboHead     fboMutexLevel = 2
```

Comment:
(Answered in chat, but also answering here for posterity.) Unfortunately, it's not. Without the explicit types, the subsequent constants are untyped which basically means they behave like ints.

The relevant part of the spec is from https://golang.org/ref/spec#Constant_declarations :

```
If the type is omitted, the constants take the individual types of the corresponding expressions. If the expression values are untyped constants, the declared constants remain untyped and the constant identifiers denote the constant values.
...
Within a parenthesized const declaration list the expression list may be omitted from any but the first declaration. Such an empty list is equivalent to the textual substitution of the first preceding non-empty expression list and its type if any. Omitting the list of expressions is therefore equivalent to repeating the previous list. The number of identifiers must be equal to the number of expressions in the previous list. Together with the iota constant generator this mechanism permits light-weight declaration of sequential values...
```

Which explains why they're untyped in general, but are actually typed in, e.g.:

```
const (
  Foo MyType = iota
  Bar
  Baz
)
```